packages/driver/cypress/integration/e2e/multi-domain/snapshots/multi_domain_snapshot_actions.spec.ts

```diff
@@ -0,0 +1,624 @@
+import _ from 'lodash'
```

Comment:
It might. A lot of these tests need to be renamed since they contain `multi-domain` all over the place. Might be a good place to include in the same effort to get our tests name compliant?

Reply:
updated in [e1af413](https://github.com/cypress-io/cypress/pull/20949/commits/e1af413b1352fc3ed768f48114df3e6594be7423)